### PR TITLE
Add Cyborg Emote Sounds

### PR DESCRIPTION
**Pull request by Keer-Sar**
## About the PR

Adds machine sounds to some cyborg emotes
## Why / Balance

Cyborgs need more content to make them more interesting.
## Technical details

 - .yml file modification only.
 Adds twpbeep.ogg and slowclap.ogg from [/tg/station](https://github.com/tgstation/tgstation)
## Media

https://github.com/space-wizards/space-station-14/assets/144283718/b0e6cf91-00b6-4466-9a08-196b9ce035b9

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## ToDo:
- [X] Add Beep
- [X] Add Chime
- [X] Add Buzz
- [X] Add Buzzes twice
- [X] Add Honk
- [X] Add Ping
- ~Add Slowclap~

## Breaking changes

**Changelog**

🆑 
- add: Cyborgs now have audio for some emotes.

### CODE REVIEW

**Comment by baddiePRO** [comment]
"oh good. my slow clap processor made it into this thing."

**Comment by EmoGarbage404** [comment]
the cyborgs having a honk sound is kinda weird

**Comment by Zap527** [comment]
Both slow clap and honk feels out of place to me.

Slow clap just isn't a sound I can see a borg making. Sure, soundboard, but then why specifically slow clap and not the others? And the honking is probably something that should be reserved to the clowning module.

File: Resources/Prototypes/Voice/speech_emotes.yml [hunk 1]
```diff
@@ -264,3 +277,139 @@
...
+    - claps slowely!
+    - clapped slowely
+    - clapped slowely.
+    - clapped slowely!
+    - clapping slow
+    - clapping slow.
+    - clapping slow!
+    - clap slow
+    - clap slow.
+    - clap slow!
```
Comment (Zap527) [inline Resources/Prototypes/Voice/speech_emotes.yml:415]: Minor spelling mistake (slowely -> slowly)

**Comment by superjj18** [comment]
Possibly outside scope but on the topic of Borg sounds, I wonder if they should have similar music functionality as pAIs have.  This pr is wonderful

File: Resources/Audio/Machines/attributions.yml [hunk 2]
```diff
@@ -152,3 +152,24 @@
...
+- files: ["buzz-sigh.ogg"]
+  license: "CC-BY-SA-3.0"
+  copyright: "Taken from TG station."
+  source: "https://github.com/tgstation/tgstation/blob/d4f678a1772007ff8d7eddd21cf7218c8e07bfc0/sound/machines/buzz-sigh.ogg"
+
+- files: ["buzz-two.ogg"]
+  license: "CC-BY-SA-3.0"
+  copyright: "Taken from TG station."
+  source: "https://github.com/tgstation/tgstation/blob/d4f678a1772007ff8d7eddd21cf7218c8e07bfc0/sound/machines/buzz-two.ogg"
+
```
Comment (metalgearsloth) [inline Resources/Audio/Machines/attributions.yml:175]: Put it in the same attribution and just link the commit.

Comment (metalgearsloth) [reply Resources/Audio/Machines/attributions.yml]: It should be a yaml list not one string, either 
[] syntax or 
\- entry 1 
\- entry 2
syntax

**Comment by LeIronGoat** [comment]
> > the cyborgs having a honk sound is kinda weird
> 
> I don't see why not to have it in, seems pretty harmless. Main reason I added it was because cyborgs and AI could do it in ss13.

Not suggesting one way or another - but something to consider for discussion is that there's a "clowning module" with a bike horn,

**Pull request merged**